app/jobs/two_factor_authentication_removal_job.rb

```diff
@@ -21,7 +21,7 @@ def perform
               DeletePublisherChannelJob.perform_now(channel_id: channel.id)
             end
           end
-          publisher.uphold_connection.disconnect_uphold
+          publisher.uphold_connection&.disconnect_uphold
```

Comment:
Unintentional commit?

Reply:
Intentional commit, wanted to fix this error 
https://github.com/brave-intl/publishers/pull/1896/commits/239ed2662389c4e1e9e10f463832f64373957811

Thought it might be quick and easy if i just did it here 🤔